### DISCUSSION TITLE
quick setup deployment error on local vm

### DISCUSSION THREAD

**Discussion by jagannathamkrishna** [egovernments/Digit-Core#84] [DIGIT Platform]
After go run digit_setup.go i am getting this error
Thanks
------------------------------------ DEPLOYING CLUSTER CONFIGS ------------------------------------
2023/05/16 16:06:58 helm template -f /home/krishna/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config.yaml --output-dir /tmp/helm-230921531 -f /home/krishna/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config-secrets.yaml .
2023/05/16 16:06:59 exit status 1: error: resource mapping not found for name: "root-ingress" namespace: "egov" from "ingress/root-ingress.yaml": no matches for kind "Ingress" in version "extensions/v1beta1"
ensure CRDs are installed first

panic: exit status 1: error: resource mapping not found for name: "root-ingress" namespace: "egov" from "ingress/root-ingress.yaml": no matches for kind "Ingress" in version "extensions/v1beta1"
ensure CRDs are installed first

**Comment by nikhilmulinti** [comment]
Hi @jagannathamkrishna , Can you please use this branch kubernetes-1.22-changes to deploy the helm charts? Do let me know incase the same issue occurs again

**Reply by jagannathamkrishna** [reply]
Hi Nikhil,
Thanks for the reply, the issue is resolved, but i am facing a new issue in post deployment when: kubectl get pods -A 
![Screenshot from 2023-05-23 10-51-24](https://github.com/egovernments/Digit-Core/assets/133766435/6ea88f1f-3456-443f-8d3b-e2943e9e36e1) getting this as per documentation(pgr-services are in crashloopbackoff state) in postman after running seed data collection 
![Screenshot from 2023-05-23 11-00-47](https://github.com/egovernments/Digit-Core/assets/133766435/bed83502-7e0c-428b-8c86-858df51cd2a1) this error is coming
In Network inspect
![Screenshot from 2023-05-23 10-55-53](https://github.com/egovernments/Digit-Core/assets/133766435/bf644293-164b-4f46-b07f-b5effa22df07)  error 500 is coming in egov-mdms-service.

thanks for the help

**Comment by jagannathamkrishna** [comment]
Thanks Egovernments/Digit-Core for the help.

The error: no match kind "Ingress" in version "extensions/v1beta1" is
resolved by deploying the master branch itself.
But facing a new issue in postman, the egov-mdms-service is throwing error
500.
Even after deleting the pods, they are still in crashloopbackoff  only.
Can this be fixed by deploying branch kubernetes-1.22-changes

Thanks in advance. Regards

On Tue, 23 May 2023 at 10:45, nikhilmulinti-egov ***@***.***>
wrote:

> Hi @jagannathamkrishna <https://github.com/jagannathamkrishna> , Can you
> please use this branch kubernetes-1.22-changes to deploy the helm charts?
> Do let me know incase the same issue occurs again
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/egovernments/Digit-Core/discussions/84#discussioncomment-5973990>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A74R2I5BR7W5GY5EXYQTFZDXHRBVTANCNFSM6AAAAAAYDO5MII>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Thanks and Best Regards,*
*Krishna Chaitanya. J*
* DevOps Consultant*
*+91-9949447178*
 https://www.perpetualblock.io
[image: Perpetual Block AG]

**Reply by shashankdhwaniris** [reply]
i am also facing the same issue while deploying the application on server, some pods are constantly crashing even after deleting the pods

**Reply by naresh-egov** [reply]
@jagannathamkrishna Kindly initiate a pod restart. Once the reboot is completed, please provide the pod description by using the '**kubectl describe pod <pod name>**' command. This will aid us in further debugging
